addons/project_todo/static/src/views/todo_activity_wizard/todo_activity_wizard_controller.js

```diff
@@ -0,0 +1,14 @@
+/** @odoo-module **/
+
+import { onMounted } from "@odoo/owl";
+import { FormController } from "@web/views/form/form_controller";
+
+export class TodoActivityWizardController extends FormController {
+    setup() {
+        super.setup();
+        onMounted(() => {
+            const firstInput = document.querySelector('div.o_field_widget input');
+            firstInput.focus();
+        });
```

Comment:
Doesn't the attribute `default_focus="1"` work in the xml?

Reply:
It does not work, because the default_focus attribute only works on new records, and the wizard model record is not considered as new (this part I'm not sure why. Maybe it's because we create the record directly from the code?). Also I forgot to remove the default_focus attribute thanks for telling me :+1: